assemblers/org.osbuild.qemu

```diff
@@ -546,6 +546,46 @@ def install_grub2(image: str, pt: PartitionTable, options):
                 grub2_write_boot_image(boot_f, image_f, core_loc)
 
 
+def parse_blsfile(blsfile):
+    params = {}
+    with open(blsfile, "r") as bls:
+        for line in bls:
+            key, value = line.split(' ', 1)
+            params[key] = value.strip()
+    return params
+
+
+def find_kernel(root):
```

Comment:
Hm.... Is there some BLS logic that dictates which entry should be chosen by default? We basically want to emulate BLS here, right? Just because zipl doesn't support it.

Reply:
It is out of the scope for the BLS:
> If userspace can figure out the available boot options, then this is only useful so much: we'd still need to come up with a way how userspace could communicate to the boot loader the default boot loader entry temporarily or persistently. Defining a common scheme for this is certainly a good idea, but out of focus for this specifications. 

But @martinezjavier explained to me that grub2 and zipl use "he rpm sorting algorithm: rpmvercmp()"

Reply:
"cool". Could we implement the same logic?

Reply:
I will file an issue about that once this is merged (so I can reference the actual code).